assets/src/js/admin/reports/components/donor-item/utils/index.js

```diff
@@ -1,20 +1,21 @@
-export function getBGColor (name) {
-    const palette = [
-        '#D75A4B',
-        '#F49420',
-        '#69B868',
-        '#556E79',
-        '#9EA3A8',
-    ]
-    return palette[Math.floor(Math.random() * (palette.length))]
+export function getBGColor() {
+	const palette = [
+		'#69B868',
+		'#556E79',
+		'#9EA3A8',
+		'#4BB5D7',
+		'#F49420',
+		'#D75A4B',
+		'#914BD7',
+	];
+	return palette[ Math.floor( Math.random() * ( palette.length ) ) ];
 }
 
-export function getInitials (name) {
-    const names = name.split(' ')
+export function getInitials( name ) {
+	const names = name.split( ' ' );
 
-    if (names.length > 1) {
-        return names[0].charAt(0) + names[names.length - 1].charAt(0)
-    } else {
-        return names[0].charAt(0)
-    }
-}
\ No newline at end of file
+	if ( names.length > 1 ) {
+		return names[ 0 ].charAt( 0 ) + names[ names.length - 1 ].charAt( 0 );
+	}
+	return names[ 0 ].charAt( 0 );
+}
```

Comment:
You might be interested in a declarative approach:
```js
export function getInitials(name) {
  return names.trim().split(' ').map(singleName => singleName.chartAt(0)).join('');
}
```

Reply:
good thinking! implemented, ref: https://github.com/impress-org/givewp/pull/4505/commits/4d7970c8ee70ec824a3b3bf8e724179861695377